### PR TITLE
Slides of metamodeling the water temperature

### PR DESCRIPTION
**Pull request by mbaudin47**
Slides from EDF's metamodeling the water temperature

### CODE REVIEW

File: README.rst [hunk 1]
```diff
@@ -49,6 +49,7 @@ OpenTURNS Presentations
   - `Surrogate-base based system reliability analysis applied to space-variant problems modeled by random-fields <https://github.com/openturns/openturns.github.io/blob/master/presentation/master/JU_OT_Amrane>`_
   - `Sensitivity analysis and uncertainty in CFD simulations of multiphase flow <https://github.com/openturns/openturns.github.io/blob/master/presentation/master/JU_OT_chenry.pdf>`_
   - `Interfacing a Modelica modeling tool and Persalys with OpenTURNS for the analysis of a solar collector <https://github.com/openturns/openturns.github.io/blob/master/presentation/master/modelica_persalys.pdf>`_
+  - `Metamodeling the water temperature <https://github.com/openturns/openturns.github.io/blob/master/presentation/master/frasilOTUsers2021.pdf>`_
```
Comment (jschueller) [inline README.rst:52]: could you put it after the release highlights to keep the chronological order ?

File: userday2021/frasil_EDF_OTUsers2021/.gitignore [hunk 2]
```diff
@@ -0,0 +1,9 @@
+*.aux
+*.bbl
+*.blg
+*.out
+*.log
+*.toc
+*.idx
+*.snm
+*.nav
```
Comment (jschueller) [inline userday2021/frasil_EDF_OTUsers2021/.gitignore:9]: The root gitignore should already have everything

File: README.rst (hunk 1 above)
Comment (mbaudin47) [reply README.rst]: Right. Fixed.

File: userday2021/frasil_EDF_OTUsers2021/.gitignore (hunk 2 above)
Comment (mbaudin47) [reply userday2021/frasil_EDF_OTUsers2021/.gitignore]: This is true. Fixed.

**Pull request merged**